pkg/controller/daemon/daemon_controller.go

```diff
@@ -94,7 +94,7 @@ type DaemonSetsController struct {
 	burstReplicas int
```

Comment:
there are still a few `context.TODO` in pkg/controller/daemon/update.go and in this file too